Backend/src/main/java/ktsnvt/tim1/controllers/EventController.java

```diff
@@ -37,4 +36,21 @@
             return new ResponseEntity<>("Dates of event days are in invalid format", HttpStatus.BAD_REQUEST);
         }
     }
+
+    @PutMapping()
+    public ResponseEntity<Object> editEvent(@Valid @RequestBody EventDTO event) {
+        try {
+            return new ResponseEntity<>(eventService.editEvent(event), HttpStatus.OK);
+        }
+        catch (EntityNotFoundException e) {
```

Comment:
Exception message should be set in service and used as e.getMessage() in controller